auth-foundation/src/main/java/com/okta/authfoundation/credential/DefaultCredentialIdDataStore.kt

```diff
@@ -38,6 +39,7 @@ internal class DefaultCredentialIdDataStore(
     private val context by lazy { ApplicationContextHolder.appContext }
 
     suspend fun getDefaultCredentialId(): String? {
+        AuthFoundation.initializeStorage()
```

Comment:
Is the default credential ID not stored within the token database?  Or is it stored elsewhere?

Reply:
It's stored in this DataStore separate from the TokenDatabase